src/main/java/frc/robot/subsystems/arm/Arm.java

```diff
@@ -179,9 +180,18 @@ public void periodic() {}
   /** Cancellation function */
   public void cancelScore() {
     goToPosition(ArmPosition.STARTING);
```

Comment:
Maybe arm should stay in current position when cancelling, and it just shouldn't release the game piece